src/mscorlib/shared/Interop/Unix/System.Native/Interop.PathConf.cs

```diff
@@ -24,50 +24,7 @@ internal enum PathConfName : int
             PC_VDISABLE         = 9,
         }
 
-        /// <summary>The maximum path length for the system.  -1 if it hasn't yet been initialized.</summary>
-        private static int s_maxPath = -1;
-
-        /// <summary>The maximum name length for the system.  -1 if it hasn't yet been initialized.</summary>
-        private static int s_maxName = -1;
-
-        internal static int MaxPath
-        {
-            get
-            {
-                // Benign race condition on cached value
-                if (s_maxPath < 0) 
-                {
-                    // GetMaximumPath returns a long from PathConf
-                    // but our callers expect an int so we need to convert.
-                    long temp = GetMaximumPath();
-                    if (temp > int.MaxValue)
-                        s_maxPath = int.MaxValue;
-                    else
-                        s_maxPath = Convert.ToInt32(temp);
-                }
-                return s_maxPath; 
-            }
-        }
-
-        internal static int MaxName
-        {
-            get
-            {
-                // Benign race condition on cached value
-                if (s_maxName < 0)
-                {
-                    int result = PathConf("/", PathConfName.PC_NAME_MAX);
-                    s_maxName = result >= 0 ? result : DEFAULT_PC_NAME_MAX;
-                }
-                
-                return s_maxName;
-            }
-        }
-
         [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_PathConf", SetLastError = true)]
         private static extern int PathConf(string path, PathConfName name);
-
-        [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_GetMaximumPath")]
-        private static extern long GetMaximumPath();
```

Comment:
do we need to remove the implementation of this method from the native side?

Reply:
I think this one is in corefx.